index.rst

```diff
@@ -1067,6 +1081,8 @@ Additional Resources
    :hidden:
    :caption: Extending PyTorch
 
+   advanced/python_custom_ops
```

Comment:
I would add "PyTorch Custom Operators Landing Page" to the "Introduction to PyTorch" list.